tools/rewriter/GenCallAsm.cpp

```diff
@@ -184,6 +189,19 @@ static void emit_reg_pop(AsmWriter &aw, const ParamLocation &loc) {
   }
 }
 
+// Emit code to copy `byte_count` bytes from `src` to `dst` using `scratch` as a
+// temporary register.
+static void emit_memcpy(AsmWriter &aw, unsigned byte_count, const std::string &dst,
```

Comment:
It's probably worth calling out that this copies a rounded-up multiple of 8-byte chunks rather than `byte_count` exactly.

Reply:
It now copies the exact number of bytes.